index.js

```diff
@@ -1,114 +1,138 @@
-var electron = require('electron')
-var argv = require('minimist')(process.argv.slice(2))
 var fs = require('fs')
 var path = require('path')
-var pkg = require('./package.json')
 
+var electron = require('electron')
 var app = electron.app
 var BrowserWindow = electron.BrowserWindow
 
 var wargs = require('./lib/args')
 var markdownToHTMLPath = require('./lib/markdown')
 
-var input = argv._[0] || argv.i || argv.input
-var output = argv._[1] || argv.o || argv.output
+var HTML_DPI = 96
 
-if (argv.v || argv.version) {
-  console.log('v' + pkg.version)
-  process.exit(0)
-}
+var PDFExporter = function (input, output, argv) {
```

Comment:
Fixes #73 by getting the CLI/child process stuff out of the module entry point.  other file named `charge.js` because it "charges" a single electron export and then dies.  Someone could now use this module to create many exports inside the same process.